### PR TITLE
Created RoleCard Component

### PR DESCRIPTION
**Pull request by rinagobo**
- Added RoleCard.js
- Added StyledRoleCard.js
- Added the code below in UserOnboardingModal.js
`<RoleCard
          iconImg={"Onboarding"} 
          title={"Developer"} 
          subtitle={"You are here to work on our codebase"} 
          theme={"theme1"}
 />`

### CODE REVIEW

File: apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/RoleCard.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { RoleCardContainer, PicWrapper, TextContents, IconImg} from './StyledRoleCard'
+
+// @description This component renders the RoleCard Component.
```
Comment (judeamos) [inline apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/RoleCard.js:4]: you can comment in multiple lines using `/** @description  ....comment */`

File: apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/RoleCard.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+export default function RoleCard({
+    iconImg,
+    title,
+    subtitle,
+    theme,
+}) {
+    return (
+        <RoleCardContainer>
+
+            <PicWrapper>
```
Comment (judeamos) [inline apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/RoleCard.js:17]: Rename this  styled component to 'IconWrapper', this explains the code better

File: apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/StyledRoleCard.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            </svg> 
+            )  
+      }
+}
+
+export const RoleCardContainer = styled.div`
+    margin-top: 50px;
+    /* Components */
```
Comment (judeamos) [inline apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/StyledRoleCard.js:24]: Not 100% sure if this is the right type of comments for this, but if your not receiving any errors, then this is fine. could you double check in the console: so when you run the app, inspect the code through the browser, then head to the console, any errors printed? if not this is fine

File: apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/StyledRoleCard.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+export const TextWrapper = styled.div`
+    text-align: center;
+    width: 100%;
+`;
+
+export const TextContents = (props) => {
+    switch (props.theme) {
+        case 'theme1':
+            return (
+                <TextWrapper>
```
Comment (judeamos) [inline apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/StyledRoleCard.js:52]: This works fine., but is there a way to refine this condition to be cleaner? so we're not duplicating the  `<Typography  style={....}/> .... <Typography />` x2 for each case.

File: apps/user-profile/src/components/modules/UserOnboardingModal/UserOnboardingModal.js
Comment (judeamos) [inline apps/user-profile/src/components/modules/UserOnboardingModal/UserOnboardingModal.js:1]: Can you revert your changes. There's no need to push this into development/user-profile branch. As the role card is not meant to be shown here. This should only be in your local/task branch.

**Review by judeamos — COMMENTED** [review]
I've reviewed your work. Can you revert the UserOnboardingModal.js back to the default version thats synced with development/user-profile branch. Once done let me know.

File: apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/RoleCard.js (hunk 1 above)
Comment (rinagobo) [reply apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/RoleCard.js]: Completed.

File: apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/RoleCard.js (hunk 2 above)
Comment (rinagobo) [reply apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/RoleCard.js]: Completed.

File: apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/StyledRoleCard.js (hunk 3 above)
Comment (rinagobo) [reply apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/StyledRoleCard.js:23]: There was no error.

File: apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/StyledRoleCard.js (hunk 4 above)
Comment (rinagobo) [reply apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/StyledRoleCard.js]: Added classes for each theme dynamically.

File: apps/user-profile/src/components/modules/UserOnboardingModal/UserOnboardingModal.js
Comment (rinagobo) [reply apps/user-profile/src/components/modules/UserOnboardingModal/UserOnboardingModal.js:1]: Deleted the RoleCard Component from UserOnboardingModal.js.

**Review by judeamos — APPROVED** [review]

**Pull request merged**